back/engines/free/email_campaigns/app/models/email_campaigns/campaigns/user_digest.rb

```diff
@@ -67,9 +67,7 @@ def generate_commands(recipient:, time: nil)
       discover_projects = discover_projects @users_to_projects[recipient.id]
 
       @notifications_counts ||= notifications_counts
-      @top_ideas ||= top_ideas.select do |idea|
-        idea.participation_method_on_creation.include_data_in_email?
-      end
```

Comment:
Remove the filtering here, as we now filter the ideas (to remove ideas that are native survey responses) beforehand, via the `TrendingIdeaService`